src/features/rewards/modalActivity/index.tsx

```diff
@@ -58,6 +58,8 @@ export interface SummaryItem {
 
 export interface Props {
   contributeRows: ContributeRow[]
+  donationRows: ContributeRow[]
```

Comment:
we don't use `donation` phrase anymore. We only have AC, tip and recurring tip

Reply:
changed